.github/workflows/ci.yml

```diff
@@ -1,5 +1,5 @@
 name: CI
```

Comment:
Why this file has been changed in the PR? 

In addition, looking to https://github.com/telefonicaid/iotagent-json/pull/527 I don't see similar changes